### PR TITLE
Minor refactoring for RaptorSplitManager

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

**Comment by nileema** [comment]
This is in preparation of a larger refactoring that we need to do for `StorageManager`

File: presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorPageSinkProvider.java [hunk 1]
```diff
@@ -38,13 +40,17 @@
...
 
     @Inject
-    public RaptorPageSinkProvider(StorageManager storageManager, PageSorter pageSorter, JsonCodec<ShardInfo> shardInfoCodec)
+    public RaptorPageSinkProvider(StorageManager storageManager, PageSorter pageSorter, JsonCodec<ShardInfo> shardInfoCodec, StorageManagerConfig config)
     {
         this.storageManager = checkNotNull(storageManager, "storageManager is null");
         this.pageSorter = checkNotNull(pageSorter, "pageSorter is null");
         this.shardInfoCodec = checkNotNull(shardInfoCodec, "shardInfoCodec is null");
+        this.maxBufferSize = config.getMaxBufferSize();
+        this.maxRows = config.getMaxShardRows();
```
Comment (electrum) [inline presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorPageSinkProvider.java]: This is not used

File: presto-raptor/src/main/java/com/facebook/presto/raptor/backup/BackupModule.java [hunk 2]
```diff
@@ -59,10 +60,16 @@ protected void setup(Binder binder)
             }
             else {
                 binder.install(module);
+                bindBackupServices(binder);
```
Comment (electrum) [inline presto-raptor/src/main/java/com/facebook/presto/raptor/backup/BackupModule.java]: Make this `binder.install(new BackupServicesModule())`

File: presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java [hunk 3]
```diff
@@ -241,14 +236,7 @@ private void writeShard(UUID shardUuid)
...
 
-    @Override
-    public PageBuffer createPageBuffer()
-    {
-        return new PageBuffer(maxBufferSize.toBytes(), Integer.MAX_VALUE);
-    }
-
-    @Override
-    public boolean isBackupAvailable()
+    private boolean isBackupAvailable()
```
Comment (electrum) [inline presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java]: This is only used in one place, inline it?

**Comment by electrum** [comment]
Looks good

**Pull request closed without merging**